src/components/dashboard/main-content/course-enrollments/course-cards/unarchive-modal/ModalBody.jsx

```diff
@@ -13,16 +13,14 @@ const ModalBody = () => {
     <>
       {confirmError && <ModalError />}
       <p className="m-0">
-        Are you sure you want to unarchive this course
+        Are you sure you want to unarchive
         {' '}
-        <a href={courseLink}>{courseTitle}</a>?
+        <a href={courseLink}>{courseTitle}</a>
         {' '}
```

Comment:
What is `{' '}` doing btw? I can't tell if it's just adding a blank newline or somehow using variable replacement in a very obtuse way.

Reply:
Great question! Yes it is basically the replacement for what we use d to as ```&nbsp;``` in older days. I think either nbsp or even `` based strings are also okay. I thikn this creates an extra text node, haven't validated on the browser...

Reply:
Pretty sure it does create an extra text node using `{' '}`. Definitely feel free to update if it makes things cleaner.